LondonBritishLibrary/orient/BLorient637.xml

```diff
@@ -50,9 +50,116 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <locus from="54r" to="63v"/>
+                                            2, stub after 8
+                                            9, stub after 1
+                                        </item>
+                                        <item xml:id="q11" n="11">
+                                            <num value="22">፳፪</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="64r" to="71v"/>
+                                        </item>
+                                        <item xml:id="q12" n="12">
+                                            <num value="23">፳፫</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="72r" to="79v"/>
+                                        </item>
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="80r" to="87v"/>
+                                        </item>
+                                        <item xml:id="q14" n="14">
+                                            <num value="25">፳፭</num>
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="88r" to="93v"/>
+                                        </item>
+                                        <item xml:id="q15" n="15">
+                                            <num value="26">፳፮</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="94r" to="101v"/>
+                                        </item>
+                                        <item xml:id="q16" n="16">
+                                            <num value="27">፳፯</num>
+                                            <dim unit="leaf">5</dim>
+                                            <locus from="102r" to="106v"/>
+                                            2, stub after 4
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
                             </supportDesc>
                         </objectDesc>
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#2r"/>
+                                    <desc type="LandGrant">Deeds of gifts, lists of furniture, books, etc., of the reigns of <persName ref="PRS9153TaklaHa"/>, <persName ref="PRS9142TaklaGi"/>
+                                        and <persName ref="PRS5618IyasuII"/>.</desc>
```

Comment:
I would say: Three records concerning land possessions and gifts, from the time of...., in two crude and one calligraphic hands. (unless you do not want to treat them separately)